src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2628,6 +2630,36 @@ def managed_db_restore(
         parameters=kwargs)
 
 
+def managed_db_restore_ltr_backup(
+        cmd,
+        client,
+        database_name,
+        managed_instance_name,
+        resource_group_name,
+        backup_id,
+        target_managed_database_name,
+        target_managed_instance_name=None,
+        target_resource_group_name=None,
+        **kwargs):
+    '''
+    Restores a managed db from an LTR backup (i.e. create with 'RestoreLongTermRetentionBackup' create mode.)
+    '''
+
+    kwargs['location'] = _get_managed_instance_location(
+        cmd.cli_ctx,
+        managed_instance_name=managed_instance_name,
```

Comment:
shouldn't this be managed_instance_name=target_managed_instance_name?

Reply:
yes